tools/tokengen/src/server/server_impl.rs

```diff
@@ -56,7 +56,7 @@ pub(crate) async fn get_provisioner_key_path() -> Result<Arc<PathBuf>, Box<dyn E
 
     let gateway_json_path = Path::new(&config_dir).join("gateway.json");
 
-    let gateway_json_contents = tokio::fs::read_to_string(&gateway_json_path).await?;
+    let gateway_json_contents:String = tokio::fs::read_to_string(&gateway_json_path).await?;
     let gateway_config: serde_json::Value = serde_json::from_str(&gateway_json_contents)?;
```

Comment:
suggestion: You need to run `cargo fmt` on `tokengen`, but I don’t think this change is necessary in the first place as `read_to_string` will obviously return a `String`.

Reply:
updated